editor/editor_properties.h

```diff
@@ -781,20 +781,19 @@ class EditorPropertyNodePath : public EditorProperty {
 	SceneTreeDialog *scene_tree = nullptr;
 	NodePath base_hint;
 	bool use_path_from_scene_root = false;
-	bool pointer_mode = false;
+	bool editing_node = false;
 
 	Vector<StringName> valid_types;
 	void _node_selected(const NodePath &p_path);
 	void _node_assign();
 	void _node_clear();
+	Node *get_base_node();
 
 	bool can_drop_data_fw(const Point2 &p_point, const Variant &p_data, Control *p_from) const;
 	void drop_data_fw(const Point2 &p_point, const Variant &p_data, Control *p_from);
 	bool is_drop_valid(const Dictionary &p_drag_data) const;
 
-	String _get_meta_pointer_property() const;
 	virtual Variant _get_cache_value(const StringName &p_prop, bool &r_valid) const override;
-	virtual StringName _get_revert_property() const override;
```

Comment:
Just realised while working on something else that what I was doing previously of returning `get_edited_property` is in fact the non-overriden behaviour so I just removed the whole overriding. Moreover I think this was the only place where this function was overriden so I'm wondering wether we should remove it to use `get_edited_property` everywhere it is used instead?

Reply:
I'm also wondering if we could remove the overriding of `_get_cache_value` but I'm not sure at 100% why there is a check for `p_prop == get_edited_property` but I think it was to update the cache when the property was being changed between pointer mode or not.